frontend/src/services/objectService.ts

```diff
@@ -26,7 +27,10 @@ export default {
     axiosOptions?: AxiosRequestConfig
   ) {
     const config = {
-      headers: { 'Content-Type': 'multipart/form-data' },
+      headers: {
+        'Content-Type': 'application/octet-stream',
+        'Content-Disposition': setDispositionHeader(object.name)
+      },
```

Comment:
Two things:
1. Please alphabetize the headers - Disposition should come before Type
2. We default to mimetype `application/octet-stream`, but we can't say every file is of this type. Please take a look at the `file-type` npm package or equivalent to have it inspect the incoming arrayBuffer stream and determine what kind of mimetype it is.